torch/csrc/profiler/python/init.cpp

```diff
@@ -326,5 +330,165 @@ void initPythonBindings(PyObject* module) {
   installCapturedTracebackPython();
 }
 
+/* [NOTE: RecordFunctionFast]
+ * These are an alternate way to call record_function from python.
+ * The typical context manager is slow (~15us on benchmarks in Aug 2023), which
+ * is usually fine for module-level annotations in python, but slow for per-op
+ * annotations. Part of the reason it is slow is because the calls go through
+ * the dispatcher, in order to make the record_function calls work with
+ * torchscript.
+ *
+ * This implementation is less safe; it doesn't work with torchscript.
```

Comment:
Can you clarify what less safe means here?
Should we simply say that this doesn't go through the dispatcher and thus won't work with any feature relying on the dispatcher (such as TS or compile).